index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Portfolio</title>
+    <meta charset="UTF-8">
+    <meta name="description" content="Adryan Corcione portfolio">
+</head>
+<body>
+    <h1 style="background-color: #C2ADFF">Adryan Corcione</h1>
+    <section id="about"">
+    <h2>About</h2>
+    <p>I'm a journalist and content marketer learning how to code. I hope to incorporate my love of pop culture into programming projects.</p>
+    </section>
```

Comment:
You could indentate this since lives within the about section, the same for the content above.